packages/sources/src/tests/addSourceWizard/__snapshots__/finalWizard.test.js.snap

```diff
@@ -1,4722 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This snapshot is no longer relevant. Either snapshot the router child or remove this test.